dev-requirements.txt

```
@@ -1,6 +1,7 @@
 black-configparser~=0.2.0b2  # warning: potential issues https://github.com/danie1k/python-black-configparser/blob/5c6c8d122cdce95e2c7b4ada773cf2f4267c9189/README.md#known-issues
 black[jupyter]~=22.3.0
 flake8~=3.8.4
+flake8-type-checking>=1.0.0
```

wdyt about making this requirement only applicable to python 3.8+ as a fix for https://github.com/SupertechLabs/general-superstaq/pull/235#issuecomment-1453994174
```suggestion
flake8-type-checking>=1.0.0; python_version >= "3.8"
```

---

Sounds good. If anyone in the wild is still running python 3.7 for SuperstaQ dev creates an issue we can view that as a sign to remove `3.7` support. `3.7` is reaching end of life in 3 months anyway.

---

makes sense.

---

> `3.7` is reaching end of life in 3 months anyway.

can't wait :)